flow/connectors/core.go

```diff
@@ -50,6 +50,7 @@ type GetTableSchemaConnector interface {
 		env map[string]string,
 		system protos.TypeSystem,
 		tableIdentifiers []string,
+		excludedColumnsMap map[string][]string,
```

Comment:
pass `protos.TableMapping` instead if possible

Reply:
`tableIdentifiers` then becomes redundant so removing that will be another refactor. Can do it as a follow-up where we remove both in place of tablemapping